enterprise/static/enterprise/admin/enterprise_customer.js

```diff
@@ -0,0 +1,18 @@
+(function($){
+    $(document).ready(function(){
+        var $catalog_anchor = $("#catalog-details-link"),
+            // URL template with have a place holder '{catalog_id}'
+            // Here is a sample url template http://localhost:18381/admin/catalogs/catalog/{catalog_id}/change/
+            url_template = $catalog_anchor.data("urlTemplate"),
+            selected_catalog = 0;
+
+        $("[name='catalog']").on("change", function(event){
+            selected_catalog = $(event.target).val();
+            $catalog_anchor.attr(
+                "href", url_template.replace("{catalog_id}", selected_catalog)
+            ).text(
+                "View catalog '{catalog_name}' details.".replace("{catalog_name}", $(event.target).find("option:selected").text())
```

Comment:
Can we eliminate `None` from displaying as an anchor ? 

<img width="416" alt="screen shot 2016-12-21 at 5 16 26 pm" src="https://cloud.githubusercontent.com/assets/7334669/21389036/536e1a6e-c7a1-11e6-8892-80445ab707e5.png">